### DISCUSSION TITLE
Component: Mentions

### DISCUSSION THREAD

**Discussion by ajitzero** [primefaces/community#788] [PrimeNG] [answered]
### Describe the feature you would like to see added

Mentions would be a directive used on input/textarea elements to configure triggers like `@`, `#` or `/` for mentioning people, linking tags, or triggering custom slash commands.

### Is your feature request related to a problem?

No.

### Describe the solution you'd like

Prime React has this already: https://primereact.org/mention/

A popular Angular variant is https://github.com/dmacfarlane/angular-mentions, but we should implement our version, which aligns itself with the React component's API as much as possible.

Unlike React, where this is a component, we should make a directive here. Then, we wouldn't need to reimplement/expose native features like the hidden property, etc., which the React component currently does.

Another feature to support here would be to support multi-character triggers, like "tag:" instead of "#", which can be considered during the current design, but can be taken up separately.

### Describe alternatives you have considered

https://github.com/dmacfarlane/angular-mentions

### Additional context

Moved this from Issues to Discussions based on https://github.com/primefaces/primeng/issues/13406#issuecomment-1871809305. Originally requested on Aug 1, 2023.

Source: primefaces/primeng#13406

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thanks for the feedback, we've added it to our internal backlog.

**Reply by Logic-Bits** [reply]
Hey @cetincakiroglu , is there already a update / ETA about this feature? Thanks!

**Reply by cagataycivici** [reply]
Not planned for near future, there are other areas that need more attention on the roadmap. [PrimeReact](https://primereact.org/mention/) has it btw.
